Makefile

```diff
@@ -109,7 +117,7 @@ clean:
 	@rm -f *.html *.xml *.out *.test
 
 .PHONY: all
-all: metalint test-ci-unit
+all: metalint test-ci-unit test-ci-integration
```

Comment:
may want to add coverage reports for this integration tests when you move to codecov